jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/deploy/SamDeployTest.kt

```diff
@@ -182,8 +185,12 @@ class SamDeployTest {
                     it.stackName(stackName)
                 }
             } catch (e: Exception) {
-                println("Failed to delete stack $stackName: ${ExceptionUtil.getMessage(e)}")
```

Comment:
Again, it was like this so we could know something is wrong with our IAM permissions. No one reads the idea.log in CodeBuild....